HW_3/task_1.py

```diff
@@ -0,0 +1,12 @@
+# Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+my_list = [1, 2, 3, 5, 2, 17, 1, 17, 4, 2, 9, 5, 6]
+res_list = []
+
+for item in my_list:
+    if my_list.count(item) > 1 and item not in res_list:
+        res_list.append(item)
```

Comment:
У вас получается квадратичная сложность. Можно немного уменьшить, при этом убрав 2-е условие из 8 строки.
```suggestion
my_list = [1, 2, 3, 5, 2, 17, 1, 17, 4, 2, 9, 5, 6]
res_list = []

for item in set(my_list):
    if my_list.count(item) > 1:
        res_list.append(item)
```